controllers/api-articles.controllers.js

```diff
@@ -0,0 +1,33 @@
+const {CheckArticleExists, fetchArticleById, readArticles, fetchComments} = require('../models')
+
+function getArticleById(req, res, next){
+    fetchArticleById(req.params.article_id)
+    .then((article) => {
+        res.status(200).send({article})
+    })
+    .catch(next)
+}
+
+function getArticles(req, res, next){
+    readArticles()
+    .then(({rows: articles}) => {
+        for (const article of articles)
```

Comment:
This is quite unusual. We simply want to return the array we received from the model (this is another taks but I had to mention!)

Reply:
Aaah I was getting an error (despite my tests passing), pretty sure that's where it came from - thanks!